src/develop/pixelpipe_hb.h

```diff
@@ -93,13 +89,10 @@ typedef struct dt_dev_pixelpipe_t
   float iscale;
   // dimensions of processed buffer
   int processed_width, processed_height;
-  // sensor saturation, propagated through the operations:
-  float processed_maximum[4];
 
-  // sensor pattern aka filters, propagated through the operations:
-  uint32_t filters; // Bayer demosaic pattern
-  /* filter for Fuji X-Trans images, only used if filters == 9u */
-  uint8_t xtrans[6][6];
+  // this one actually contains the expected output format,
+  // and should be modified by process*(), if necessary.
+  dt_iop_buffer_dsc_t dsc;
```

Comment:
Well, right now it is there for 2 main reasons
- I thought you suggested putting it there
- And, right now, with processed_max (and filters), all the iops modify the object in this struct

As for tiling, right now it works exactly like it did without the change.
processed_max is averaged when leaving tiling.
But filters are problematic indeed.
There is this tiling->xalign and tiling->xalign, and it currently 2.0, so for bayer we should be good.
But for X-Trans...